engineering-plans/0001-graphql-query-prefetching.md

```diff
@@ -0,0 +1,397 @@
+# PLAN-0001: GraphQL Query Prefetching
+
+<dl>
+  <dt>Author</dt>
+  <dd>David Lutterkort</dd>
+
+<dt>Implements</dt>
+<dd>No RFC - no user visible changes</dd>
+
+<dt>Engineering Plan pull request</dt>
+<dd><a href="<https://github.com/graphprotocol/rfcs/pull/2>"><https://github.com/graphprotocol/rfcs/pull/2></a></dd>
```

Comment:
This links is broken ever so slightly. Should be
```
<a href="https://github.com/graphprotocol/rfcs/pull/2">https://github.com/graphprotocol/rfcs/pull/2</a>
```

Reply:
Too much copy pasta. Fixed